src/main/java/htsjdk/variant/variantcontext/GenotypeBuilder.java

```diff
@@ -28,6 +28,7 @@
 import htsjdk.tribble.util.ParsingUtils;
 import htsjdk.variant.vcf.VCFConstants;
```

Comment:
Add disclaimer at the top of this class: `Re-using the same GenotypeBuilder to build multiple Genotype objects via calls to make() is dangerous, since reference types in the builder (eg., Collections/arrays) don't get copied when making each Genotype. To safely re-use the same builder object multiple times, use makeWithShallowCopy() instead of make().`